LondonBritishLibrary/orient/BLorient7945.xml

```diff
@@ -0,0 +1,165 @@
+                            <note>The vocables are arranged in the order of the Latin alphabet. In <locus from="45v" to="50r"/> the order is:
+                                transcription, English translation, the word written in Amharic. In <locus from="51r" to="89r"/> the order is:
+                                the word written in Amharic and English translation (without transcription of the Amharic). In this vocabulary the English
+                                and the transcription is written in black, the Amharic in red.</note>
+                            <textLang xml:lang="en" otherLangs="am"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="89v" to="180r"/>
+                            <title ref="NAR0184ForeignWords">Short English sentences with Amharic translation</title>
+                            <note>The sentences are arranged according to the key words in the order of the Latin alphabet. These English words and 
+                                the Amharic text are written in red, the rest of the sentence in black, i.e.: 
+                                <foreign xml:lang="en">It is <hi rend="rubric">about</hi> five cubits long</foreign>
+                                <foreign xml:lang="am"><hi rend="rubric">አምስት፡ ክንድ፡ ርጅም፡ የምያህል፡</hi></foreign>.</note>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">92</measure>
+                                    <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                                        <height>232</height>
+                                        <width>190</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Original pagination by the author from 1 to 180.</foliation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="23"><locus target="#2v"/><locus from="45v" to="50r"/>
+                                    <locus from="89v" to="180r"/></layout>
+                                <layout columns="2" writtenLines="23"><locus from="3r" to="44r"/><locus from="50v" to="89r"/></layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Latin">
+                                <desc>Hand of the author <persName role="author" ref="PRS8943Speedy">
```

Comment:
you can encode: `<handNote xml:id="h1" script="Latin">` and also 
 `<seg type="ink">Black; red.</seg>`  "... written in black and red ink" will not be neccessary in this case.